astrogrism/tests/test_image2grism.py

```diff
@@ -71,36 +73,39 @@ def test_wfc3_grismconf(grism, grism_image):
                                          astrogrism_x,
                                          astrogrism_y,
                                          grism_image)
-    np.testing.assert_allclose(roundtrip_wavelengths, grismconf_wavelengths)
+    assert_quantity_allclose(roundtrip_wavelengths, grismconf_wavelengths)
```

Comment:
Out of curiosity, what was the reasoning behind moving from `numpy.testing` to `astropy.tests.helper`? Not questioning the decision, just genuinely curious

Reply:
`assert_quantities_allclose` lets you compare `Quantities` and take care of units, `assert_allclose` does not.